samples/attested_tls/scripts/gen_mrenclave_header.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Open Enclave SDK contributors.
```

Comment:
@radhikaj @anakrish this was originally a bash script and I ported it to python so it could run as part of both the Windows and Linux builds.

In general, is there a preference for what language should be used for cross-platform build scripts? Ideally we settle on just one so we don't have too many build-time dependencies.